chia/util/streamable.py

```diff
@@ -126,6 +126,10 @@ def is_type_Tuple(f_type: object) -> bool:
     return get_origin(f_type) is tuple or f_type is tuple
 
 
+def is_type_Dict(f_type: object) -> bool:
+    return get_origin(f_type) is dict or f_type is dict
```

Comment:
should this handle `Mapping` or such?

Reply:
I will leave the _should_ to you but it _doesn't_ as currently implemented.